client/components/food-entries.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+class FoodEntries extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: []
+    };
+    this.sumCalories = this.sumCalories.bind(this);
```

Comment:
You don't have to bind this method, since it is not being passed as an event handler or callback function.

Reply:
![Screen Shot 2021-04-14 at 4 57 36 PM](https://user-images.githubusercontent.com/74885918/114795030-8e8deb80-9d42-11eb-9fd7-c987b07b430d.png)